test/README.md

```diff
@@ -147,13 +147,13 @@ export GCP_SERVICE_ACCOUNT_KEY_PATH="$PWD/config.json"
 
 ### Running
 
-Integration tests live in this directory. To run these tests, you must provide
+End to end tests live in this directory. To run these tests, you must provide
 `go` with `-tags=e2e`. By default the tests run against your current kubeconfig
 context, but you can change that and other settings with [the flags](#flags):
 
 ```shell
-go test -v -count=1 -tags=e2e ./test
-go test -v -tags=e2e -count=1 ./test --kubeconfig ~/special/kubeconfig --cluster myspecialcluster
+go test -v -count=1 -tags=e2e -timeout=20m ./test
+go test -v -count=1 -tags=e2e -timeout=20m ./test --kubeconfig ~/special/kubeconfig --cluster myspecialcluster
```

Comment:
I believe tests e2e tests already take more then 20m and growing, can we change it to 30

Reply:
They are all running in parallel so adding more tests should not result in 10m increase of test timing.

Reply:
:sweat:

Reply:
They already take about 25m
https://gubernator.knative.dev/builds/knative-prow/pr-logs/pull/knative_build-pipeline/473/pull-knative-build-pipeline-integration-tests/